src/libs/stackdriverReporter.js

```diff
@@ -42,7 +42,11 @@ export const StackdriverReporter = {
     const user = Storage.getCurrentUser();
     const formattedMsg = await StackdriverReporter.format(msg);
     errorHandler.setUser(ld.get(user, 'email', 'anonymous'));
-    errorHandler.report(formattedMsg);
+    try {
+      await errorHandler.report(formattedMsg);
+    } catch (error) {
+      //
```

Comment:
I think it's worth adding a comment here for why we're swallowing the error:

```suggestion
      // swallow error to avoid user visible errors
```